From 5feec0e92b7bc0fcf8180dcc6d28699efd19609e Mon Sep 17 00:00:00 2001
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Mon, 2 Jul 2018 15:30:49 +0100
Subject: [PATCH 1305/1666] ASoC: sof: loader: cleanup - add comments and
 reorder funcs

Add comments describing funcs/code blocks and reorder funcs so features
are grouped.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
(cherry picked from commit 81063c85e9ed7f63175547607572dfc6094e5eaa)
---
 sound/soc/sof/loader.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sound/soc/sof/loader.c b/sound/soc/sof/loader.c
index 82d6c6c..4bc7d7a 100644
--- a/sound/soc/sof/loader.c
+++ b/sound/soc/sof/loader.c
@@ -6,6 +6,8 @@
  * Copyright(c) 2017 Intel Corporation. All rights reserved.
  *
  * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *
+ * Generic firmware loader.
  */
 
 #include <linux/delay.h>
@@ -42,6 +44,7 @@ static int get_ext_windows(struct snd_sof_dev *sdev,
 	return ret;
 }
 
+/* parse the extended FW boot data structures from FW boot message */
 int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, u32 offset)
 {
 	struct sof_ipc_ext_data_hdr *ext_hdr;
@@ -279,5 +282,6 @@ EXPORT_SYMBOL(snd_sof_run_firmware);
 
 void snd_sof_fw_unload(struct snd_sof_dev *sdev)
 {
+	/* TODO: support module unloading at runtime */
 }
 EXPORT_SYMBOL(snd_sof_fw_unload);
-- 
2.7.4

